assets/dev/js/editor/document/commands/base/base.js

```diff
@@ -18,9 +25,15 @@ export default class Base extends ArgsObject {
 		// Acknowledge self about which command it run.
 		this.currentCommand = $e.commands.getCurrent( 'document' );
 
+		// Assign instance of current component.
+		this.component = $e.commands.getComponent( this.currentCommand );
+
 		// Who ever need do something before without `super` the constructor can use `initialize` method.
 		this.initialize( args );
 
+		// You can edit `this.args` from `initialize`.
```

Comment:
(Don't use "You" in docs...)
```suggestion
		// Refresh arg, maybe they via the `initialize`.
```